orion-server/src/main/java/com/pinterest/orion/core/actions/aws/AmiTagManager.java

```diff
@@ -0,0 +1,100 @@
+package com.pinterest.orion.core.actions.aws;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.pinterest.orion.server.api.AMI;
+
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.services.ec2.Ec2Client;
+import software.amazon.awssdk.services.ec2.model.DescribeImagesRequest;
+import software.amazon.awssdk.services.ec2.model.DescribeImagesResponse;
+import software.amazon.awssdk.services.ec2.model.CreateTagsRequest;
+import software.amazon.awssdk.services.ec2.model.CreateTagsResponse;
+import software.amazon.awssdk.services.ec2.model.Filter;
+import software.amazon.awssdk.services.ec2.model.Tag;
+
+public class AmiTagManager {
+  private static final Logger logger = Logger.getLogger(AmiTagManager.class.getCanonicalName());
+  private Ec2Client ec2Client;
+
+  public AmiTagManager() {
+    ec2Client = Ec2Client.create();
+  }
+
+  public List<AMI> getAmiList(Map<String, String> filter) {
+    List<AMI> amiList = new ArrayList<>();
+    DescribeImagesRequest.Builder builder = DescribeImagesRequest.builder();
+    builder = builder.filters(
+      Filter.builder().name("tag:application").values("kafka").build()
```

Comment:
Can we please use constant variables for all this fixed strings across the board?

Reply:
Done